tests/test_augmentations.py

```diff
@@ -221,7 +228,8 @@ def test_augmentations_wont_change_input(augmentation_cls, params, image, mask):
 def test_augmentations_wont_change_float_input(augmentation_cls, params, float_image):
     float_image_copy = float_image.copy()
     aug = augmentation_cls(p=1, **params)
-    aug(image=float_image)
+    annotations = {'image': float_image, 'cropped_bbox': [-59, 77, 177, 231]}
```

Comment:
you should not change tests, just add another if existing does not fit your needs.